tests/Meziantou.Framework.TypeConverter.Tests/DefaultConverterTests_DateTimeTo.cs

```diff
@@ -10,7 +10,7 @@ public class DefaultConverterTests_DateTimeTo
     public void TryConvert_DateTimeToDateTimeOffset()
     {
         var converter = new DefaultConverter();
-        var cultureInfo = CultureInfo.CurrentCulture;
+        var cultureInfo = new CultureInfo("en-US");
```

Comment:
Using `new CultureInfo(name)` returns a new editable instance. The static method uses another ctor which returns a readonly instance. It also caches the instances.

```suggestion
        var cultureInfo = CultureInfo.GetCultureInfo("en-US");
```